README.md

```diff
@@ -26,18 +26,39 @@ See our [LTS policy](https://github.com/vtex-apps/awesome-io#lts-policy) for mor
 
 This app uses our store builder with the blocks architecture. To know more about Store Builder [click here.](https://help.vtex.com/en/tutorial/understanding-storebuilder-and-stylesbuilder#structuring-and-configuring-our-store-with-object-object)
 
-To use this app, you need to import in your dependencies on `manifest.json`.
+This app is used as a block in our [Store Header](https://github.com/vtex-apps/store-header/blob/master/store/interfaces.json). You can configure the minicart properties 
+
+To configure or customize this app, you need to import in your dependencies on `manifest.json`.
 
 ```json
   dependencies: {
     "vtex.minicart": "2.x"
   }
 ```
 
-Then, add `minicart` block into our app theme, as we do in our [Dreamstore app](https://github.com/vtex-apps/dreamstore/blob/master/store/blocks.json). 
+Then, add `minicart` block into your app theme as we do in our [Dreamstore app](https://github.com/vtex-apps/dreamstore/blob/master/store/blocks.json). 
```

Comment:
Why there is the `Store Header` example and the `Dreamstore` one?

Reply:
Store Header is the app that adds minicart block. Dreamstore is where we configure the minicart behavior.